src/null_platform.h

```diff
@@ -29,13 +29,13 @@
 
 #define _GLFW_PLATFORM_WINDOW_STATE _GLFWwindowNull null
 
-#define _GLFW_PLATFORM_CONTEXT_STATE
-#define _GLFW_PLATFORM_MONITOR_STATE
-#define _GLFW_PLATFORM_CURSOR_STATE
-#define _GLFW_PLATFORM_LIBRARY_WINDOW_STATE
-#define _GLFW_PLATFORM_LIBRARY_CONTEXT_STATE
-#define _GLFW_EGL_CONTEXT_STATE
-#define _GLFW_EGL_LIBRARY_CONTEXT_STATE
+#define _GLFW_PLATFORM_CONTEXT_STATE         struct { int dummy; }
+#define _GLFW_PLATFORM_MONITOR_STATE         struct { int dummy; }
+#define _GLFW_PLATFORM_CURSOR_STATE          struct { int dummy; }
+#define _GLFW_PLATFORM_LIBRARY_WINDOW_STATE  struct { int dummy; }
+#define _GLFW_PLATFORM_LIBRARY_CONTEXT_STATE struct { int dummy; }
+#define _GLFW_EGL_CONTEXT_STATE              struct { int dummy; }
+#define _GLFW_EGL_LIBRARY_CONTEXT_STATE      struct { int dummy; }
```

Comment:
This causes name conflict errors because more than one of these goes into the library struct.

Reply:
Done